core/src/com/unciv/ui/components/widgets/ZoomableScrollPane.kt

```diff
@@ -263,13 +260,16 @@ open class ZoomableScrollPane(
             onZoomStopListener?.invoke()
         }
 
-        override fun zoom(initialDistance: Float, distance: Float) {
-            if (lastInitialDistance != initialDistance) {
-                lastInitialDistance = initialDistance
-                lastScale = scaleX
-            }
-            val scale: Float = sqrt((distance / initialDistance).toDouble()).toFloat() * lastScale
-            zoom(scale)
+        override fun zoom(delta: Float, focus: Vector2) {
+            // correct map position to zoom in to the focus point
+            val dx = (stage.width / 2 - focus.x) * (delta - 1) / scaleX
+            val dy = (stage.height / 2 - focus.y) * (delta - 1) / scaleY
+            scrollTo(
+                scrollX - dx,
+                scrollY + dy,
```

Comment:
Is this difference due to a difference between screen and stage coordinates, regarding top-left or top-right at 0,0?